### PR TITLE
Moving files issue3097

### PR DESCRIPTION
**Pull request by eduartua**
Realted to #3097 

Moving files that need to be updated and it is better they live in contributors/design-proposals/architecture. 

/sig contributor-experience
/kind feature

/assign @nikhita @bgrant0607 
/cc @parispittman

### CODE REVIEW

File: contributors/design-proposals/architecture/architecture.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Kubernetes Design and Architecture
 
 A much more detailed and updated [Architectural
-Roadmap](../../devel/architectural-roadmap.md) is also available.
+Roadmap](architectural-roadmap.md) is also available.
```
Comment (nikhita) [inline contributors/design-proposals/architecture/architecture.md]: Can you change the link to ` /contributors/design-proposals/architecture/architectural-roadmap.md`?

Comment (eduartua) [reply contributors/design-proposals/architecture/architecture.md]: Done

**Comment by eduartua** [comment]
Also, related to #3064

**Comment by cblecker** [comment]
@eduartua Are you able to `git mv` these files instead, so we retain the git history on them?

You can also use this local function to always git mv things when you're in a git repo:
```
function mv() {
    if git rev-parse --git-dir >/dev/null 2>&1; then
        git mv "$@"
    else
        mv "$@"
    fi
}
```

**Comment by eduartua** [comment]
@cblecker sure, thank you.

**Comment by eduartua** [comment]
> @eduartua Are you able to `git mv` these files instead, so we retain the git history on them?
> 
> You can also use this local function to always git mv things when you're in a git repo:
> 
> ```
> function mv() {
>     if git rev-parse --git-dir >/dev/null 2>&1; then
>         git mv "$@"
>     else
>         mv "$@"
>     fi
> }
> ```

Done

**Pull request merged**